### PR TITLE
Liftoff: Update contact email address

### PR DESCRIPTION
**Pull request by Vungle-GordonTian**
update contact email address

### CODE REVIEW

**Comment by Vungle-GordonTian** [comment]
@gargcreation1992 PTAL of this PR.

File: static/bidder-info/liftoff.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: "https://rtb.ads.vungle.com/bid/t/c770f32"
 maintainer:
-  email: platform-ssp@liftoff.io
+  email: vxssp@liftoff.io
```
Comment (onkarvhanumante) [inline static/bidder-info/liftoff.yaml:3]: @Vungle-GordonTian  Prebid team has sent an mail on `vxssp@liftoff.io` for maintainer email verification

Comment (Vungle-GordonTian) [reply static/bidder-info/liftoff.yaml:3]: Sent. Please check.

Comment (onkarvhanumante) [reply static/bidder-info/liftoff.yaml:3]: > Response sent. Please check.

received response

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**